src/query/ast/src/parser/statement.rs

```diff
@@ -999,10 +998,7 @@ pub fn insert_source(i: Input) -> IResult<InsertSource> {
             let rest_str = &rest_tokens[0].source
```

Comment:
There is a special case that rest_tokens may be empty, so it's not safe to call `rest_tokens[0]`  

See tests in :  https://github.com/datafuselabs/databend/pull/7524/files#diff-777d8f965ebb763eb535886aba615867aab230a5c9caea59851822777fa5d523R10-R14